gradle.properties

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright 2020 Daniel Scherf & Michael Rittmeister & Julian K�nig
+# Copyright 2021 Daniel Scherf & Michael Rittmeister & Julian K�nig
```

Comment:
Do you know the german term vErScHlImMbEsSeRn? Maybe try UTF-8 in all files?

Reply:
I thought our global encoding was set to UTF-8